### PR TITLE
Create docs join-as-a-mainnet-validator-using-snapshots-chainlayer.md

### PR DESCRIPTION
**Pull request by satawatnack**

### CODE REVIEW

File: docs/joining-mainnet/join-as-a-mainnet-validator-using-snapshots.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+sudo apt-get update -y
+sudo apt-get install wget liblz4-tool aria2 -y
+sudo apt install -y jq
+```
+Download snapshots
+```
+cd ~/.band/
+
+# change network to default/pruned/archive depending on your needs
+URL=`curl -L https://quicksync.io/band.json|jq -r '.[] |select(.file=="laozi-mainnet-pruned")|.url'`
```
Comment (RogerKSI) [inline docs/joining-mainnet/join-as-a-mainnet-validator-using-snapshots.md]: validator can choose the type of snapshot (pruned/default). think need to give information more a bit.

File: docs/joining-mainnet/join-as-a-mainnet-validator-using-snapshots.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+aria2c -x5 $URL
+wget https://raw.githubusercontent.com/chainlayer/quicksync-playbooks/master/roles/quicksync/files/checksum.sh
+wget $URL.checksum
+
+# Compare checksum with onchain version. Hash can be found at $URL.hash
+curl -s https://lcd-cosmos.cosmostation.io/txs/`curl -s $URL.hash`|jq -r '.tx.value.memo'|sha512sum -c
```
Comment (RogerKSI) [inline docs/joining-mainnet/join-as-a-mainnet-validator-using-snapshots.md]: this lcd-cosmos still works, right? last time, it seems can't get the hash.

**Review by RogerKSI — CHANGES_REQUESTED** [review]

File: docs/joining-mainnet/join-as-a-mainnet-validator-using-snapshots-chainlayer.md [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+## Step 2: Download Snapshots
+
+Install compression tools
+```
+sudo apt-get update -y
+sudo apt-get install wget liblz4-tool aria2 -y
+sudo apt install -y jq
+```
+Download snapshots
+You can choose `laozi-mainnet-pruned` size 144G or `laozi-mainnet-default` size 1.8T
```
Comment (taobun) [inline docs/joining-mainnet/join-as-a-mainnet-validator-using-snapshots-chainlayer.md:114]: 100-200 GB (prune) and ~2TB

**Review by taobun — CHANGES_REQUESTED** [review]

**Comment by colmazia** [comment]
Does we gonna continue doing p’Bun idea mai na, about unifying all of validator-joining docs

**Review by colmazia — COMMENTED** [review]
Does we gonna continue doing p’Bun idea mai na, about unifying all of validator-joining docs

**Comment by RogerKSI** [comment]
As we decided to use tabs to reduce duplicated content, I have moved the content here with the tab section to the new PR. please review there instead (https://github.com/bandprotocol/bandchain-docs/pull/125).

**Pull request closed without merging**